usaspending_api/awards/v2/urls_idv_awards.py

```diff
@@ -2,9 +2,13 @@
 from usaspending_api.awards.v2.views.idvs.amounts import IDVAmountsViewSet
 from usaspending_api.awards.v2.views.idvs.awards import IDVAwardsViewSet
 from usaspending_api.awards.v2.views.idvs.funding import IDVFundingViewSet
+from usaspending_api.awards.v2.views.idvs.funding_treemap import IDVFundingRollupViewSet  # , IDVFundingTreemapViewSet
 
 urlpatterns = [
     url(r'^amounts/(?P<requested_award>[A-Za-z0-9_. -]+)/$', IDVAmountsViewSet.as_view()),
     url(r'^awards/$', IDVAwardsViewSet.as_view()),
     url(r'^funding/$', IDVFundingViewSet.as_view()),
+    url(r'^funding-rollup/$', IDVFundingRollupViewSet.as_view()),
+    # This is a stub for the tree map endpoint fo DEV-2237
+    # url(r'^funding-treemap/$', IDVFundingTreemapViewSet.as_view()),
```

Comment:
Original plan for the treemap endpoint is to be `/api/v2/awards/idvs/accounts/ `